obs_common/waitfor.py

```diff
@@ -28,6 +46,9 @@ def main(args=None):
     )
     parser.add_argument("--verbose", action="store_true")
     parser.add_argument("--timeout", type=int, default=15, help="Wait timeout")
```

Comment:
I documented this poorly--it's very confusing. What's really going on is that waitfor will keep attempting with a 5-second timeout until this timeout parameter is exceeded.

At some point, we should either rename this argument or fix the documentation. We don't have to do that in this PR.